samples/drivers/charger/src/main.c

```diff
@@ -32,6 +32,28 @@ int main(void)
 
 	printk("Found device \"%s\", getting charger data\n", chgdev->name);
 
+	val.discharge_current_notification.current_ua = 5000000;
```

Comment:
Should such values be Kconfigs, and are these default values ok to have? To put it differently: are these hard coded value realistically (and safely?) applicable to any setup and type of battery the person testing will be using?

Reply:
yeah that sounds like a better as more high power chargers get merged in. I'll add some Kconfig options.

Reply:
I think I'm close here (a1b86115c544d2632eb1baa8b589ce45a2bb9de9), but I am running into a few issue with the config symbols. I'll try and resolve it on the next push.